src/Explorer/Controls/InfiniteProgressBar/InfiniteProgressBarComponent.less

```diff
@@ -0,0 +1,57 @@
+.progress {
```

Comment:
This is great, but can't we use this?
https://developer.microsoft.com/en-us/fluentui#/controls/web/progressindicator

Reply:
The progressindicator looks better! added that instead